SnapWarShareExtension/SWSharePageViewController.swift

```diff
@@ -0,0 +1,150 @@
+import UIKit
+
+class SWSharePageViewController: UIPageViewController {
```

Comment:
the reason i use page view controller is because it has native support for a bunch of things such as swiping through the pages and showing the position where im at in the carousel via the dots at the bottom. all i needed to do was implement some of the delegate and data source methods to make this work!